plugins/netmask/plugin.go

```diff
@@ -0,0 +1,67 @@
+package netmask
+
+import (
+	"encoding/binary"
+	"errors"
+	"net"
+
+	"github.com/coredhcp/coredhcp/handler"
+	"github.com/coredhcp/coredhcp/logger"
+	"github.com/coredhcp/coredhcp/plugins"
+	"github.com/insomniacslk/dhcp/dhcpv4"
+	"github.com/insomniacslk/dhcp/dhcpv6"
+)
+
+var log = logger.GetLogger()
+
+func init() {
+	plugins.RegisterPlugin("netmask", setupNetmask6, setupNetmask4)
+}
+
+var (
+	netmask net.IPMask
+)
+
+func setupNetmask6(args ...string) (handler.Handler6, error) {
+	log.Printf("plugins/netmask: loaded plugin for DHCPv6.")
```

Comment:
please add a TODO/Warningf/issue here to say that this is not implemented yet for v6